lib/libspl/include/sys/file.h

```diff
@@ -37,10 +37,10 @@
 
 #define FCREAT  O_CREAT
 #define FTRUNC  O_TRUNC
-#define FOFFMAX 0
+//#define FOFFMAX 0
```

Comment:
Just a thought, but leaving FOFFMAX as 0 lets us get away with not needing #ifdef APPLE around each place that uses it...

Reply:
The idea was to tidy up by not (mis-)defining things, but I am not completely wedded to the idea of doing #ifdef __APPLE__ instead of defining FOFFMAX.